apps/pie-storybook/.storybook/preview.js

```diff
@@ -16,4 +16,16 @@ export default {
             },
         },
     },
+    parameters: {
+      a11y: {
+        config: {
+          rules: [
+            {
+              id: 'WCAG Rules',
+              tags: ['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria', 'wcag21aaa']
```

Comment:
Super formatting NIT from me:

```suggestion
              tags: [
                  'wcag21a',
                  'wcag21aa',
                  'wcag143',
                  'cat.color',
                  'cat.aria',
                  'wcag21aaa'
              ]
```